File: in_Python/0461 Hamming Distance.py
class Solution(object):
    def hammingDistance(self, x, y):
        """
        :type x: int
        :type y: int
        :rtype: int
        """
        res = 0
        while x != 0 and y != 0:
            a = x % 2
            b = y % 2
            if a != b: 
                res += 1
            x /= 2
            y /= 2
        while x == 0 and y != 0:
            res += y % 2
            y /= 2
        while x != 0 and y == 0:
            res += x % 2
            x /= 2    
        return res

File: in_Java/0461 Hamming Distance.java
class Solution {
    public int hammingDistance(int x, int y) {
        int res = 0;
        while(x != 0 && y != 0){
            if(x % 2 != y % 2) res++;
            x /= 2;
            y /= 2;
        }
        while(x != 0){
            res += x % 2;
            x/=2;
        }
        while(y != 0){
            res += y % 2;
            y/=2;
        }
        return res;
    }
}